### ISSUE TITLE
running script/bootstrap results in error and script stops

### ISSUE THREAD

**Issue by quiltykim** [education/classroom#1348]
##
#### What you were trying to do ?
_I tried to run script/bootstrap on macos_

#### What happened (include screenshot if you can) ?
_Received "Error: Unknown command: bootstrap-rbenv-ruby"_

#### Step-by-step reproduction instructions
See above

**Comment by Kadaaran** [comment]
Hi @quiltykim 👋 

Why were you trying to run `script/bootstrap` on itself ? 🤔 
To run the app you just need to run `script/setup` then `script/server` , does it work on your end ?

Thanks !

**Comment by quiltykim** [comment]
Hi @Kadaaran 👋 

Thanks for the response. I actually found out about this repo through this lecture: https://youtu.be/NrEdRi4U2Zk?t=278  and I suppose I took John's advice about "the only prerequisite [being] docker" too literally and ran `script/bootstrap` without consulting the readme.

But yes, running `script/setup` followed by `script/server` got me to where I need to be on macos just fine.

On Arch Linux, I noticed that I needed to install ngrok and gnu-netcat for nc, which one of the scripts calls for. I'm not too sure how edgy this use-case is but thought I'd bring it up.

Thanks again!
